scripts/generate_panelite_content.py

```diff
@@ -17,15 +17,54 @@
+    "gallery/layout/dynamic_tabs.ipynb": ["https://github.com/holoviz/panel/issues/4393"],
+    "gallery/layout/plot_with_columns.ipynb": ["https://github.com/holoviz/panel/issues/4393"],
+    "gallery/param/action_button.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/param/deck_gl_global_power_plants.ipynb": ["https://github.com/holoviz/panel/issues/4612"],
+    "gallery/param/download_upload_csv.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/param/loading_indicator.ipynb": ["https://github.com/holoviz/panel/issues/4613"],
+    "gallery/param/precedence.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/param/reactive_plots.ipynb": ["https://github.com/holoviz/panel/issues/4393", "https://github.com/holoviz/panel/issues/4416"],
+    "gallery/param/reactive_tables.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/simple/altair_brushing.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/simple/clifford_interact.ipynb": ["https://github.com/holoviz/datashader/issues/1200"],
+    "gallery/simple/color_speech_recognition.ipynb": ["https://github.com/holoviz/panel/issues/4404"],
+    "gallery/simple/deckgl_game_of_life.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/simple/defer_data_load.ipynb": ["https://github.com/holoviz/panel/issues/4393", "https://github.com/holoviz/panel/issues/4416"],
+    "gallery/simple/file_download_examples.ipynb": ["https://github.com/apache/arrow/issues/34996"],
+    "gallery/simple/hvplot_explorer.ipynb": ["https://github.com/holoviz/panel/issues/4393", "https://github.com/holoviz/panel/issues/4416"],
+    "gallery/simple/iris_kmeans.ipynb": ["https://github.com/holoviz/panel/issues/4393", "https://github.com/holoviz/panel/issues/4416"],
+    "gallery/simple/loading_spinner.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/simple/temperature_distribution.ipynb": ["https://github.com/holoviz/panel/issues/4393"],
+    "gallery/simple/xgboost_classifier.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/streaming/hardware_automation.ipynb": ["https://github.com/holoviz/panel/issues/4393"],
+    "gallery/streaming/hardware_automation.ipynb": ["https://github.com/holoviz/panel/issues/4393"],
+    "gallery/streaming/streaming_indicator.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/streaming/streaming_perspective.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/styles/PlotlyStyle.ipynb": ["https://github.com/holoviz/panel/issues/4416"],
+    "gallery/styles/SeabornStyle.ipynb": ["https://github.com/holoviz/panel/issues/4615"],
+    "gallery/viz/GraphViz.ipynb": ["https://github.com/xflr6/graphviz/issues/197"],
+    "gallery/viz/NetworkX.ipynb": ["https://github.com/pygraphviz/pygraphviz/issues/453"],
```

Comment:
All of these are being moved and/or removed. I'll update the PR when I'm done with that.